test_pytest_html.py

```diff
@@ -174,18 +174,34 @@ def test_create_report_path(self, testdir):
         assert result.ret == 0
         assert_results(html)
 
+    def test_resources_inline_css(self, testdir):
+        testdir.makepyfile('def test_pass(): pass')
+        result, html = run(testdir, 'report.html',
+                           '--self-contained-html', 'True')
+        assert result.ret == 0
+        for resource in ['style.css', 'main.js']:
```

Comment:
We only need to test the CSS here as the JavaScript is still inline by default and tested in `test_resources`.